app/brave_generated_resources.grd

```diff
@@ -1163,6 +1163,33 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
             Add selected
           </message>
         </if>
+        <message name="IDS_PLAYLIST_DEFAULT_PLAYLIST_NAME" desc="A name for the default playlist folder">
+          Play Later
+        </message>
+        <message name="IDS_PLAYLIST_MOVE_MEDIA_DIALOG_TITLE" desc="A window title for a dialog wihch is to moving media to another folder">
+          Move Media
+        </message>
+        <message name="IDS_PLAYLIST_MOVE_MEDIA_DIALOG_NEW_PLAYLIST" desc="A label for a button to create a new playlist and and move media to it">
+          New Playlist
+        </message>
+        <message name="IDS_PLAYLIST_MOVE_MEDIA_DIALOG_BACK" desc="A label for 'Back' button ">
+          Back
+        </message>
+        <message name="IDS_PLAYLIST_MOVE_MEDIA_DIALOG_OK" desc="A label for 'OK' button on move dialog">
+          Save
+        </message>
+        <message name="IDS_PLAYLIST_MOVE_MEDIA_DIALOG_CREATE_AND_MOVE" desc="A label for 'OK' button when creating a new playlist and move media">
+          Create and move
+        </message>
+        <message name="IDS_PLAYLIST_MOVE_MEDIA_DIALOG_DESCRIPTION" desc="A description for move dialog">
+          Select a folder to move this media to
+        </message>
+        <message name="IDS_PLAYLIST_MOVE_MEDIA_DIALOG_PLAYLIST_NAME" desc="A title for new playlist's name textfield">
+          Playlist name
+        </message>
+        <message name="IDS_PLAYLIST_MOVE_MEDIA_DIALOG_SELECTED_ITEMS" desc="Shows how many items are about to be moved">
+          <ph name="NUM_ITEMS">$1<ex>4</ex></ph> selected
```

Comment:
Nit: if there's a room for it, maybe `items selected` instead of just `selected`